test/test_dynamic_shapes.py

```diff
@@ -291,6 +291,17 @@ def test_size_expressions(self):
         self.assertTrue(str(expand_x.shape[1]), str(x.shape[0]))
         self.assertTrue(str(expand_x.shape[1]), str(result.shape[0]))
 
+    @skipIfNoSympy
```

Comment:
Test added for numel

Reply:
@ezyang, I suppose this is the desirable behavior: 
numel(SymbolicTensor) returns a SymInt. 
num(RealTensor) returns an int.

Reply:
Yup that sounds right